tox.ini

```diff
@@ -18,6 +18,39 @@ whitelist_externals =
 # We removed cleanup and destroy from the middle it breaks later tasks.
```

Comment:
I think this comment is now wrong. Cleanup may have been needed to be removed, but destroy needs to be there.